code/modules/mob/holder.dm

```diff
@@ -54,24 +54,6 @@ var/list/holder_mob_icon_cache = list()
 	for(var/mob/M in contents)
 		M.show_inv(usr)
 
-/obj/item/weapon/holder/proc/sync(var/mob/living/M)
```

Comment:
this sync function we imported with baymerge mostly duplicated functionality i'd already written. so it'd not being used for simple holders, code moved to the specific human subclass